src/sentry/testutils/helpers/features.py

```diff
@@ -44,6 +48,22 @@ def Feature(names):
 
     def features_override(name, *args, **kwargs):
         if name in names:
+            try:
+                feature = features.get(name, organization=None)
+            except FeatureNotRegistered:
+                logger.info("Unregistered flag defaulting to False: %s", repr(name))
+                return False
```

Comment:
I think it'd be good to throw an error here as well, just make the tests hard fail